agentarchives/archivesspace/client.py

```diff
@@ -288,15 +288,21 @@ def _format_dates(self, start, end=None):
             return start
 
     def _fetch_dates_from_record(self, record):
+        dates = self._fetch_date_expression_from_record(record)
+        if not dates:
+            try:
+                start_date = record['dates'][0]['begin']
```

Comment:
The `dates` key always exists in the record; it'll be an empty list in that case.